### PR TITLE
updates for mongo ycsb scripts 

### PR DESCRIPTION
**Pull request by ekuric**
Below files were updated 
-  external_vars.yaml
-  added recordcount and operationcount options
- files/scripts/test-mongo.sh - sorted results and write them to .csv files for easier drawing afterwards
- mongodb-test.yaml - some aditions for pbench-user-benchmark --config section
runmongo.sh - wrapper script which helps to run multiple tests in one run

With these changes we can run multiple test combinations in one run and ```runmongo.sh``` helps with jenkins parameters 

Signed-off-by: Elvir Kuric <elvirkuric@gmail.com>

### CODE REVIEW

**Comment by ekuric** [comment]
@hongkailiu @mffiedler ptal and comment where necessary

File: storage/mongodb/files/scripts/test-mongo.sh [hunk 1]
```diff
@@ -19,17 +21,33 @@ echo "NAMESPACE: ${NAMESPACE}"
...
-  done
+
+for i in $(seq 1 ${ITERATION}); do 
+  for load  in $(echo ${WORKLOAD} | sed -e s/,/" "/g); do 
+	for thread in $(echo ${THREADS} | sed -e s/,/" "/g); do 
+    		oc -n ${NAMESPACE} exec $(oc get pod -n ${NAMESPACE} | grep mongodb | awk '{print $1}') -- scl enable rh-mongodb32 -- mongo -u redhat -p redhat ${MONGODB_IP}:27017/testdb --eval "db.usertable.remove({})"
+		oc -n ${NAMESPACE} exec $(oc get pod -n ${NAMESPACE} | grep ycsb | awk '{print $1}') -- ./bin/ycsb load mongodb -s -threads $thread -P "workloads/${load}" -p mongodb.url=mongodb://redhat:redhat@${MONGODB_IP}:27017/testdb -p recordcount=${RECORDCOUNT} -p operationcount=${OPERATIONCOUNT} 2>&1 | tee -a ${output_dir}/mongodb_load_data_${load}_threads_${thread}.txt
+
+		oc -n ${NAMESPACE} exec $(oc get pod -n ${NAMESPACE} | grep ycsb | awk '{print $1}') -- ./bin/ycsb run mongodb -s -threads $thread -P "workloads/${load}" -p mongodb.url=mongodb://redhat:redhat@${MONGODB_IP}:27017/testdb 2>&1 -p recordcount=${RECORDCOUNT} -p operationcount=${OPERATIONCOUNT}  | tee -a ${output_dir}/mongodb_run_load_${load}_threads_${thread}.txt
+	done
```
Comment (hongkailiu) [inline storage/mongodb/files/scripts/test-mongo.sh]: Why do we run the same test twice here?

Comment (hongkailiu) [reply storage/mongodb/files/scripts/test-mongo.sh]: NIP: for consistency ... 2>&1 after -p recordcount=${RECORDCOUNT} -p operationcount=${OPERATIONCOUNT}

**Pull request merged**